messages/diagnostics.md

```diff
@@ -14,3 +14,7 @@ The salesforcedx plugin is deprecated. Uninstall it by running: `%s plugins:unin
 # linkedPluginWarning
 
 Warning: the [%s] plugin is linked.
+
+# uninstallSuggestion
+
+Please uninstall the Salesforce CLI (%s) version %s and install the @salesforce/cli version 2. See https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_uninstall.htm for uninstall instructions.
```

Comment:
It might be nice to tell users _why_ we're asking them to uninstall. I see we're saying "using sfdx-cli version 7" and "using @salesforce/cli version 1" in the check. Maybe this is as simple as saying... "using _deprecated_ version 7 sfdx-cli". I know there is an official definition/implication for the term deprecated, but I can't find that definition. Maybe @jshackell-sfdc has a suggestion.

Reply:
Well, we use "deprecated" all over the place in the CLI, so that ship has sailed :)    I like Ryan's suggestion (i.e. adding "deprecated" where appropriate in the initial statement.)

As far as this message, please remove "Please"  (we don't use it in our docs).  So:

Uninstall Salesforce CLI (%s) version %s and install the @salesforce/cli version 2. See https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_uninstall.htm for uninstall instructions.